shared/settings/files/index.desktop.js

```diff
@@ -0,0 +1,106 @@
+// @flow
```

Comment:
this works cross platform so likely better to not have .desktop and .native versions of this. just rename this files/index.js and use isMobile to return () => null in the default export

Reply:
I don't follow...we will need settings on native, and none of the existing settings I've implemented apply to mobile.

Reply:
the primary motivation for specific native/desktop files is if we're using a component that isn't cross platform. if you're using purely cross platform components then you can make a single file that condititionally uses isMobile to render different things. this makes the flow typing better (as it can actually type the file and not the .js.flow facade).

Reply:
Oh I see, thanks.